src/main/resources/public/js/activity.controller.js

```diff
@@ -2,19 +2,19 @@
   'use strict';
   angular.module('app').controller('ActivityController', ActivityController);
 
-  ActivityController.$inject = [ '$log', 'activityService', 'CONST', 'loginService', 'utilService' ];
+  ActivityController.$inject = ['$log', 'activityService', 'CONST', 'loginService', 'utilService'];
 
   function ActivityController($log, activityService, CONST, loginService, utilService) {
-    var logger = $log.getInstance('ActivityController');
-    var vm = this;
+    const logger = $log.getInstance('ActivityController');
+    const vm = this;
 
     // public methods
     vm.list = list;
     vm.remove = remove;
 
     // public fields
     vm.activities = {
-      totalElements: 0
+      totalElements: 0,
```

Comment:
See http://eslint.org/docs/rules/comma-dangle.html and https://github.com/airbnb/javascript#commas--dangling.